jsk_kinova_robot/jsk_kinova_description/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
please use `cmake_minimum_required(VERSION 2.8.3)` to fix 
https://github.com/jsk-ros-pkg/jsk_robot/pull/1379/checks?check_run_id=3633340176#step:7:2804

Reply:
Changed